test/playwright/utils/breakpoints.ts

```diff
@@ -56,42 +56,39 @@ const makeBreakpointDescribe =
     blockOrOptions: T,
     block?: T extends Record<string, unknown> ? BreakpointBlock : undefined
   ) => {
-    test.describe(
-      `screen at breakpoint ${breakpoint} with width ${screenWidth}`,
-      () => {
-        const _block = (
-          typeof blockOrOptions === 'function' ? blockOrOptions : block
-        ) as BreakpointBlock
-        const options =
-          typeof blockOrOptions !== 'function'
-            ? { ...defaultOptions, ...blockOrOptions }
-            : defaultOptions
-
-        test.use({
-          viewport: { width: screenWidth, height: 700 },
-          userAgent: options.uaMocking ? mockUaStrings[breakpoint] : undefined,
-        })
-
-        const getConfigValues = (name: string) => ({
-          name: `${name}-${breakpoint}.png` as const,
-        })
-
-        const expectSnapshot = async <T extends ScreenshotAble>(
-          name: string,
-          screenshotAble: T,
-          options?: Parameters<T['screenshot']>[0]
-        ) => {
-          const { name: snapshotName } = getConfigValues(name)
-          return expect(
-            await screenshotAble.screenshot(options)
-          ).toMatchSnapshot({
+    test.describe(`screen at breakpoint ${breakpoint} with width ${screenWidth}`, () => {
```

Comment:
For some reason, this was re-formatted by the linter. The main change in this file is the type of ExpectSnapshot, which was throwing an error.